torch_xla/csrc/pooling.cpp

```diff
@@ -78,11 +78,11 @@ PoolingOpAttributes MakePoolingOpAttributes(
 // from the given input_size, when the stride is the same as the kernel size.
 std::vector<xla::int64> AdaptiveAvgPoolKernelSize(
     absl::Span<const xla::int64> input_size,
-    absl::Span<const xla::int64> output_size) {
+    absl::Span<const xla::int64> output_size, int unsigned pool_dim) {
```

Comment:
consider using `xla::uint32` if you want it to be unsigned.

Reply:
Happy to make the change. I just did not know why we prefer xla::uint32 vs int unsigned?

Reply:
hmm, actually I think this one is fine as it is. When we call xla functions, they usually takes `xla::int64` or other `xla:: type`. We did not directly pass this number to xla so it is fine to leave it as it is.